haskell/private/actions/compile.bzl

```diff
@@ -212,8 +250,11 @@ def _compilation_defaults(hs, cc, java, dep_info, srcs, extra_srcs, cpp_defines,
   args.add(compiler_flags)
   haddock_args.add(compiler_flags, before_each="--optghc")
 
-  # Output static and dynamic object files.
-  args.add(["-static", "-dynamic-too"])
```

Comment:
If I understand correctly this hack is a workaround to an upstream issue in GHC. It would be good to call that out explicitly, and to look to the relevant upstream issue.

Reply:
I don't know if there is a ticket for this, actually. CC @hvr

Reply:
After some searching, opened https://ghc.haskell.org/trac/ghc/ticket/15394 for now. Even if it's a dupe, we'll know that soon.

Reply:
I added a note in the source too.